src/main/java/com/bretpatterson/schemagen/graphql/impl/GraphQLObjectMapper.java

```diff
@@ -154,10 +154,13 @@ private void buildGenericArgumentTypeMap(ParameterizedType type) {
 
 	private Optional<String> getFieldNameFromMethod(Method m) {
 		Optional<String> fieldName = Optional.absent();
-		if (m.getName().startsWith("get")) {
+
+		// Do not match 'get' method
+		if (m.getName().startsWith("get") && m.getName().length() > 3) {
```

Comment:
Simple workaround, I did not feel like creating a full regex was worth the effort here